<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>368</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>368</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0.689, y1:0.233, x2:0.578, y2:0.136273, stop:0.0945274 rgba(83, 79, 20, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Square_Shape">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(174, 149, 255);
                              border : 2px solid black
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Square_Side">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Square_Label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Enter  side of square</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Square_Submit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
border-radius : 15; 
background-color: rgb(255, 233, 188);
                              border : 2px solid black
</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
